Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -263,8 +259,8 @@ void Tileset::_updateLodTransitions(
     // Update fade out
     for (auto tileIt = result.tilesFadingOut.begin();
          tileIt != result.tilesFadingOut.end();) {
-      TileRenderContent* pRenderContent =
-          (*tileIt)->getContent().getRenderContent();
+      TileRenderContent* pRenderContent = const_cast<TileRenderContent*>(
```

Comment:
This `const_cast` I can more easily understand; we want the `Tile`s in `ViewUpdateResult` to be non-modifiable by public API users, but the `Tileset` must modify them under-the-hood to apply these transitions.

Reply:
Yeah, although this is actually pointing to a bigger problem, well outside the scope of this PR. Now that we have multiple view groups, it makes no sense for the fade percentage to be stored on the Tile.